File: platforms/web/examples/core/texture_mip_chain/format.js
import { TextureFormat, TextureMipChain } from "fragmentcolor";

const pixels = new Uint8Array(4 * 4 * 4);
pixels.fill(200);
const chain = TextureMipChain.prepare(pixels, TextureFormat.Rgba8UnormSrgb, [4, 4]);
const _ = chain.format();

File: platforms/swift/examples/texture/mipmap/format.swift
import FragmentColor
import Foundation

let pixels = Data(repeating: 200, count: 4 * 4 * 4)
let chain = try Mipmap.build(
    bytes: pixels,
    format: .rgba8UnormSrgb,
    size: Size(width: 4, height: 4, depth: nil)
)
let _ = chain.format()